pkg/observation.go

```diff
@@ -88,7 +89,8 @@ func observationsToUpkeepKeys(
 
 		// validate the observation using the provided validator
 		if err := upkeepObservation.Validate(v); err != nil {
-			logger.Printf("unable to decode observation: %s", err.Error())
+			logger.Printf("%s", string(obs.Observation))
```

Comment:
Should this log be removed, or combined with the log line immediately after it?

Reply:
removed it